include/micm/cuda/util/cuda_sparse_matrix.hpp

```diff
@@ -32,18 +32,18 @@ namespace micm
       this->param_.d_data_ = nullptr;
     }
 
-    CudaSparseMatrix(const SparseMatrixBuilder<T, OrderingPolicy>& builder)
-        : SparseMatrix<T, OrderingPolicy>(builder)
+    CudaSparseMatrix(const SparseMatrixBuilder<T, OrderingPolicy>& builder, bool empty_matrix = false)
+        : SparseMatrix<T, OrderingPolicy>(builder, empty_matrix)
```

Comment:
I was thinking after we talked about how to name these flags. What do you think about `indexing_only`? I think that might be more descriptive of what you would use the flag for.